src/consensus/aft/raft.h

```diff
@@ -564,10 +582,23 @@ namespace aft
         throw std::runtime_error(
           "learner requires two-transaction reconfiguration");
       }
+
       backup_nodes.clear();
       create_and_remove_node_state();
     }
 
+    void add_identity(
+      ccf::SeqNo seqno,
+      kv::ReconfigurationId rid,
+      const ccf::Identity& identity)
+    {
+      if (use_two_tx_reconfig && require_identity_for_reconfig)
+      {
+        assert(byzantine_identity_tracker);
```

Comment:
On :264, we check systematically, but this is just an assert here. Is this something that always needs checking at runtime or a semi-static guarantee?

Reply:
Just a sanity check - since we check and throw an exception in line 264, this assertion should always hold.